src/entities/activity/ui/items/ActionPlan/Page.tsx

```diff
@@ -0,0 +1,127 @@
+import { useContext } from 'react';
+
+import { Body } from './Body';
+import { DocumentContext } from './DocumentContext';
+import { Header } from './Header';
+import { usePageWidth, usePageMaxHeight, useXScaledDimension, usePageMinHeight } from './hooks';
+import { ActivitiesPhrasalData } from './phrasalData';
+import { Phrase } from './Phrase';
+import { Title } from './Title';
+
+import footerLogo from '~/assets/mindlogger-action-plan-footer-logo.svg';
+import { PhrasalTemplatePhrase } from '~/entities/activity/lib';
+import { useActionPlanTranslation } from '~/entities/activity/lib/useActionPlanTranslation';
+import { Theme } from '~/shared/constants';
+import Box from '~/shared/ui/Box';
+
+type PageProps = {
+  pageNumber: number;
+  phrases: PhrasalTemplatePhrase[];
+  phrasalData: ActivitiesPhrasalData;
+  appletTitle: string;
+  phrasalTemplateCardTitle: string;
+  noImage: boolean;
+};
+
+export const Page = ({
+  appletTitle,
+  phrasalTemplateCardTitle,
+  phrases,
+  phrasalData,
+  pageNumber,
+  noImage,
+}: PageProps) => {
+  const { totalPages } = useContext(DocumentContext);
+  const { t } = useActionPlanTranslation();
+  const pageWidth = usePageWidth();
+  const pageMinHeight = usePageMinHeight();
+  const pageMaxHeight = usePageMaxHeight();
+  const scaledPadding = useXScaledDimension(16);
+  const scaledTopPadding = useXScaledDimension(28);
```

Comment:
Just need to beef up the spacing at the top a tad, too. There's actually a total of 40 pixels between the edge of the border and the top of the `<Header>` (16 + 24):
![CleanShot 2024-09-10 at 18 26 49](https://github.com/user-attachments/assets/9ead4f41-437b-4097-af88-3eff56e3bd29)

```suggestion
  const scaledTopPadding = useXScaledDimension(40);
```